tests/TestCase/Command/ModelCommandTest.php

```diff
@@ -1146,9 +1107,10 @@ public function testGetValidationExcludeForeignKeysSigned()
                 'integer' => ['rule' => 'integer', 'args' => []],
                 'notEmpty' => ['rule' => 'notEmptyString', 'args' => []],
             ],
-            'id' => [
+            'user_id' => [
                 'integer' => ['rule' => 'integer', 'args' => []],
-                'allowEmpty' => ['rule' => 'allowEmptyString', 'args' => [null, 'create']],
+                'requirePresence' => ['rule' => 'requirePresence', 'args' => ['create']],
```

Comment:
We should have a test for nullable foreignKey field too and sure `requirePresence()` is not used for it.